fastlane/lib/fastlane/actions/delete_keychain.rb

```diff
@@ -22,13 +33,20 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :name,
                                        env_name: "KEYCHAIN_NAME",
                                        description: "Keychain name",
-                                       optional: false)
+                                       conflicting_options: [:path],
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :path,
```

Comment:
I know some other actions are using this name already, but I'd prefer the name to be `keychain_path` to be more explicit 👍